### PR TITLE
0.4.1rc fixes

### PR DESCRIPTION
**Pull request by mhl10**
* adds `flask_sqlalchemy` package
* updates `wsgi.py` per other repositories 

This is currently deployed on beta. There are still some problems in the staging environment that I'm investigating; there may be additional commits to this branch.

### CODE REVIEW

**Review by erickpeirson — APPROVED** [review]
Cool, thanks for catching these

**Comment by mhl10** [comment]
@erickpeirson please note latest commits related to flask-s3.

File: accounts/accounts/config.py [hunk 1]
```diff
@@ -134,3 +134,10 @@
      'minimum_ttl': 360000}
 ]
 """Requests for Vault secrets."""
+
+FLASKS3_BUCKET_NAME = os.environ.get('FLASKS3_BUCKET_NAME', 'some_bucket')
+FLASKS3_CDN_DOMAIN = os.environ.get('FLASKS3_CDN_DOMAIN', 'static.arxiv.org')
+FLASKS3_USE_HTTPS = os.environ.get('FLASKS3_USE_HTTPS', 1)
+FLASKS3_FORCE_MIMETYPE = os.environ.get('FLASKS3_FORCE_MIMETYPE', 1)
+FLASKS3_ACTIVE = os.environ.get('FLASKS3_ACTIVE', 0)
```
Comment (erickpeirson) [inline accounts/accounts/config.py]: These look good. While you're in here, can you add an ``APP_VERSION = 0.4`` so that the static paths work correctly?

Comment (mhl10) [reply accounts/accounts/config.py]: Good catch; added.

**Comment by mhl10** [comment]
@erickpeirson please feel free to merge this branch on my behalf if you're satisfied with the latest changes and the checks pass

**Review by E5624 — APPROVED** [review]

**Review by kaledbutler — APPROVED** [review]

**Review by E5624 — APPROVED** [review]

**Pull request merged**